tests/e2e/vsphere_volume_fstype.go

```diff
@@ -183,41 +187,13 @@ func invokeTestForInvalidFstype(f *framework.Framework, client clientset.Interfa
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	}()
 
-	// Waiting for PVC to be bound
-	var pvclaims []*v1.PersistentVolumeClaim
-	pvclaims = append(pvclaims, pvclaim)
-	ginkgo.By("Waiting for all claims to be in bound state")
-	persistentvolumes, err := fpv.WaitForPVClaimBoundPhase(client, pvclaims, framework.ClaimProvisionTimeout)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
-	// Create a Pod to use this PVC, and verify volume has been attached
-	ginkgo.By("Creating pod to attach PV to the node")
-	pod, err := createPod(client, namespace, nil, []*v1.PersistentVolumeClaim{pvclaim}, false, execCommand)
+	ginkgo.By(fmt.Sprintf("Expect claim to fail as filesystem %q is invalid.", fstype))
+	err = fpv.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client,
```

Comment:
instead of this we can directly poll for the error in events

Reply:
done